src/applications/check-in/day-of/pages/Confirmation/MultipleAppointments.jsx

```diff
@@ -18,16 +15,14 @@ const MultipleAppointments = props => {
 
   const appointment = selectedAppointment;
   const appointmentDateTime = new Date(appointment.startTime);
-  const focusOnLoad = useCallback(() => {
```

Comment:
This should still happen on this page but in a useEffect like on other pages:
```
useEffect(() => {
  focusElement('h1');
}, []);
```

Reply:
I will add this to this PR, but is would it be worth extracting this into possibly an HoC so that it automatically applies to every page?

Reply:
I will add this to this PR, but is would it be worth extracting this into possibly an HoC so that it automatically applies to every page?

Reply:
I like DRY, only question is HoC or custom hook? Maybe a discussion topic for Thursday... 🙂